packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/close/proposeConfig.ts

```diff
@@ -0,0 +1,45 @@
+import { MnemonicKey } from '@terra-money/terra.js'
+import { instructionToTailCommand, TailInstruction } from '../../../abstract/executionWrapper'
+import ProposeConfig, { CommandInput as ProposeConfigInput } from '../proposeConfig'
+
+type CommandInput = Pick<ProposeConfigInput, 'proposalId'>
+
+const makeInput = (flags, args): CommandInput => {
+  if (flags.input) return flags.input as CommandInput
+  return {
+    proposalId: flags.configProposal,
+  }
+}
+
+const makeInnerCommandInput = (input: CommandInput): ProposeConfigInput => {
+  const randomAcc = () => new MnemonicKey().publicKey?.address()!
```

Comment:
- [ ] Generate 4 vanity addresses that represent empty transmitters/payees

Reply:
You can try to adapt this Cosmos vanity addr generator: https://github.com/hukkin/cosmosvanity

Reply:
https://github.com/smartcontractkit/chainlink-terra/pull/239/files#diff-07ced786da08b434e12acebf624b6588e9307b71be6574b4bcf479d746b892c7R45